src/frontend/packages/lib_classroom/src/components/ClassroomForm/index.spec.tsx

```diff
@@ -447,4 +450,45 @@ describe('<ClassroomForm />', () => {
     ).toBeInTheDocument();
     expect(screen.getByText(/invite\/my-token/i)).toBeInTheDocument();
   });
+
+  it('displays a list of available recordings', () => {
+    const classroom = classroomMockFactory({ id: '1', started: false });
+    const classroomRecordings = [
+      classroomRecordingMockFactory({
```

Comment:
Should not add `classroom: classroom.id` in these 2 factories ?

Reply:
Theres two steps in this test :
the first one there's no recording displayed, and for the second one, a rerender is done, adding recordings to the classroom. : https://github.com/openfun/marsha/pull/1952/files/59cb4e0130e118f52fb37abd787ee5e7dd335059..aee7e414056a54addf3b34f6dbccd25235144ff8#diff-c2e606b95dd7b553e32197e21d192d3ecba4026e50e2d17e7582ce7500d32087R475-R482